test/unit/specs/components/EventMerge/createComponent.spec.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import createComponent from "../../../../../src/components/EventMerge/createComponent";
+
+describe("EventMerge:createComponent", () => {
+  it("creates a component", () => {
+    const createEventMergeId = () => undefined;
+    expect(createComponent({ createEventMergeId })).toEqual({
+      commands: {
+        createEventMergeId: {
+          run: createEventMergeId
+        }
+      }
+    });
+  });
+});
```

Comment:
I'm still seeing the error I mentioned in https://github.com/adobe/alloy/pull/632/files#r518533046.